src/couch_replicator_httpc.erl

```diff
@@ -169,13 +169,31 @@ process_stream_response(ReqId, Worker, HttpDb, Params, Callback) ->
 % on the ibrowse_req_id format. This just drops all
 % messages for the given ReqId on the floor since we're
 % no longer in the HTTP request.
-clean_mailbox({ibrowse_req_id, ReqId}) ->
+
+clean_mailbox(ReqId) ->
+    clean_mailbox(ReqId, 16).
```

Comment:
I contemplated that but I don't think we want to make this configurable. Not only is it super duper esoteric in its intent and purpose, but if we ever find people needing to actually change this value, the bug lies elsewhere, not here.